src/gc/gc.cpp

```diff
@@ -16311,6 +16311,7 @@ void gc_heap::update_collection_counts ()
 inline
 BOOL AnalyzeSurvivorsRequested(int condemnedGeneration)
 {
+#ifndef BUILD_AS_STANDALONE
     // Is the list active?
```

Comment:
What is the end-user visible debugger feature that is not going to work with standalone GC because of this?

Reply:
I think it's just some events are written to debugger on a gc, the only thing I could find that uses it is https://github.com/dotnet/coreclr/blob/54bdc1681c08cf9230561733d4728bc13cbc494e/src/ToolBox/SOS/Strike/strike.cpp#L6880-L6904

@noahfalk please correct me if I missed something

Reply:
That OnGcEvent method is part of the implementation for !FindRoot
https://github.com/dotnet/coreclr/blob/master/src/ToolBox/SOS/Strike/sosdocs.txt#L1918
Without this event !FindRoot -gen <number> won't ever break back into the debugger.